dnsmessage.proto

```diff
@@ -41,45 +41,47 @@ message PBDNSMessage {
     TCP = 2;                                    // Transmission Control Protocol (RFC 793)
   }
   enum PolicyType {
-    UNKNOWN = 1;                                // No policy applied, or unknown type
+    UNKNOWN = 1;                                // No RPZ policy applied, or unknown type
     QNAME = 2;                                  // Policy matched on the QName
     CLIENTIP = 3;                               // Policy matched on the client IP
     RESPONSEIP = 4;                             // Policy matched on one of the IPs contained in the answer
     NSDNAME = 5;                                // Policy matched on the name of one nameserver involved
     NSIP = 6;                                   // Policy matched on the IP of one nameserver involved
   }
-  required Type type = 1;
+  required Type type = 1;                       // Type of event
   optional bytes messageId = 2;                 // UUID, shared by the query and the response
   optional bytes serverIdentity = 3;            // ID of the server emitting the protobuf message
   optional SocketFamily socketFamily = 4;
   optional SocketProtocol socketProtocol = 5;
-  optional bytes from = 6;                      // DNS requestor (client)
-  optional bytes to = 7;                        // DNS responder (server)
+  optional bytes from = 6;                      // DNS requestor (client) as 4 (IPv4) or 16 (IPv6) raw bytes in network byte order
+  optional bytes to = 7;                        // DNS responder (server) as 4 (IPv4) or 16 (IPv6) raw bytes in network byte order
   optional uint64 inBytes = 8;                  // Size of the query or response on the wire
   optional uint32 timeSec = 9;                  // Time of message reception (seconds since epoch)
   optional uint32 timeUsec = 10;                // Time of message reception (additional micro-seconds)
   optional uint32 id = 11;                      // ID of the query/response as found in the DNS header
 
   message DNSQuestion {
-    optional string qName = 1;
-    optional uint32 qType = 2;
-    optional uint32 qClass = 3;
+    optional string qName = 1;                  // Fully qualified DNS name (with trailing dot)
+    optional uint32 qType = 2;                  // https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-4
```

Comment:
Perhaps we could specify the byte ordering for these fields as well?

Reply:
At second thought, I don't it make sense since the way numerical values are encoded is defined by the protocol buffer format.